airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/DefaultAirbyteSource.java

```diff
@@ -57,16 +54,8 @@ public DefaultAirbyteSource(final IntegrationLauncher integrationLauncher) {
   }
```

Comment:
@cgardens this is needed for the abstract source tests - did not look closely why. mainly wanted to get the build working.